lib/decidim/action_delegator/admin_engine.rb

```diff
@@ -18,6 +18,7 @@ class AdminEngine < ::Rails::Engine
           resources :permissions, only: [:create] do
             post :sync, on: :collection
           end
+          delete "remove_census", to: "participants#remove_census", as: "remove_census"
```

Comment:
Instead of this, you can reuse the `manage_participants` with the standard "destroy" method:

```ruby
resources :manage_participants, only: [:new, :create, :destroy]
```